app/views/admin/application/_navigation.html.haml

```diff
@@ -1,8 +1,10 @@
 %nav.navigation{:role => "navigation"}
   = link_to 'Application', root_path, class: 'navigation__link navigation__link--inactive'
   - Administrate::Namespace.new(namespace).resources.each do |resource|
+    - next if resource.name === :reports
```

Comment:
can use stronger assertion here `===` or `equal?`. I _think_ rubocop might catch this, though not positive since it's in a view

Reply:
I need 👓